haystack/nodes/connector/crawler.py

```diff
@@ -83,17 +86,40 @@ def __init__(
                     This example will generate a file name from the url by replacing all characters that are not allowed in file names with underscores.
                  2) crawler_naming_function=lambda url, page_content: hashlib.md5(f"{url}{page_content}".encode("utf-8")).hexdigest()
                     This example will generate a file name from the url and the page content by using the MD5 hash of the concatenation of the url and the page content.
+        :param webdriver_options: A list of options to send to Selenium webdriver. If none is provided,
+            Crawler uses, as a default option, a reasonable selection for operating locally, on restricted docker containers,
+            and avoids using GPU.
+            Crawler always appends the following option: "--headless"
```

Comment:
Why do we always need to append the `--headless` option?

Reply:
Oh, this was already default on haystack. The main reason is that without `--headless` mode, X server would need to be installed and display environment correctly setup. Basically, it will not show the UI, if applied.

Not using `--headless` mode increases the memory and disk space footprint dramatically.

Reply:
Ahh okay, I see. Sounds good to keep.